tests/Pest.php

```diff
@@ -2,9 +2,16 @@
 
 use Stancl\Tenancy\Tests\TestCase;
 
-uses(TestCase::class)->in(__DIR__);
+uses(TestCase::class)->in(...filesAndFoldersExcluding(['WithoutTenancy']));
 
-function pest(): TestCase
+function pest(): \Orchestra\Testbench\TestCase
```

Comment:
This removes IDE support for the methods we've added in our `TestCase`

Reply:
But we get the return type error when using `WithoutTenancy` TestCase.

Reply:
Looks like returning `TestCase` works now. Previously, the `WithoutTenancy` TestCase was extending `\Orchestra\Testbench\TestCase` but now I'm extending it from our `Stancl/Tests/TestCase`.